### PR TITLE
Staker fixes 5: Make assertions absolute instead of relative to the last assertion 

### PR DESCRIPTION
**Pull request by PlasmaPower**

### CODE REVIEW

**Review by joshuacolvin0 — APPROVED** [review]
LGTM

**Review by hkalodner — CHANGES_REQUESTED** [review]

File: packages/arb-node-core/staker/validator.go [hunk 1]
```diff
@@ -93,14 +93,14 @@ func (v *Validator) resolveTimedOutChallenges(ctx context.Context) (*types.Trans
...
-	confirmType, successorWithStake, stakerAddress, err := v.validatorUtils.CheckDecidableNextNode(ctx)
+	confirmType, err := v.validatorUtils.CheckDecidableNextNode(ctx)
 	if err != nil {
 		return err
 	}
 	switch confirmType {
 	case ethbridge.CONFIRM_TYPE_INVALID:
 		logger.Info().Msg("Rejecting node")
-		return v.rollup.RejectNextNode(ctx, successorWithStake, stakerAddress)
+		return v.rollup.RejectNextNode(ctx, v.wallet.Address())
```
[the remark this answers is not in the document]
Comment (PlasmaPower) [reply packages/arb-node-core/staker/validator.go:103]: I don't think you should be, but there might be an edge case, and it couldn't hurt to check.

**Comment by PlasmaPower** [comment]
Resolved comments

**Review by hkalodner — APPROVED** [review]
LGTM

**Pull request merged**